File: app/src/main/java/com/example/website/consulta/Inicio.java
package com.example.website.consulta;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.view.Window;
import android.widget.Button;

public class Inicio extends AppCompatActivity {

    Button btnconsu,btnmoto;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.activity_inicio);

        btnconsu = findViewById(R.id.btnconsu);
        btnmoto  = findViewById(R.id.btnmoto);

        btnconsu.setOnClickListener(new View.OnClickListener()
        {
            @Override
            public void onClick(View v)
            {
                startActivity(new Intent( Inicio.this,Principal.class));

            }
        });
        btnmoto.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(Inicio.this,Motores.class));
            }
        });
    }
}

File: app/src/main/java/com/example/website/consulta/View/Inicio.kt
package com.example.website.consulta.View

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.view.*
import android.widget.*
import com.example.website.consulta.R

class Inicio : AppCompatActivity() {
    var btnconsu: Button? = null
    var btnmoto: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportRequestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.activity_inicio)
        btnconsu = findViewById(R.id.btnconsu)
        btnmoto = findViewById(R.id.btnmoto)
        btnconsu?.setOnClickListener(View.OnClickListener { startActivity(Intent(this@Inicio, Principal::class.java)) })
    }
}